.github/workflows/test-pr.yml

```diff
@@ -270,7 +270,7 @@ jobs:
           k-deb-path: kframework.deb
       - name: 'Run integration tests'
         run: |
-          docker exec -u user k-pyk-integration-${{ github.sha }} make test-integration TEST_ARGS='-n4 --timeout 300 -vv'
```

Comment:
Do we want to keep `-vv`?

Reply:
I don't think so. I added that last week when trying to help out Georgy get more info from the CI failure (he was having trouble reproducing it locally). It just added a ton of not super useful timing information, see [this](https://github.com/runtimeverification/k/actions/runs/9178802879/job/25240227479#step:6:1935)

Reply:
Makes sense!